### PR TITLE
feat(alertStatus): Add Alert Conditions to Alert Status Page Sidebar

### PR DESCRIPTION
**Pull request by mikellykels**
Add Alert Conditions to Alert Status Page Sidebar.

[FIXES WOR-1613](https://getsentry.atlassian.net/browse/WOR-1613)

Screen Shot 2022-02-22 at 11 44 45 PM

**With no filters**
Screen Shot 2022-02-22 at 11 53 41 PM

### CODE REVIEW

File: static/app/views/alerts/details/sidebar.tsx [hunk 1]
```diff
@@ -17,6 +19,75 @@ type Props = {
 };
 
 class Sidebar extends PureComponent<Props> {
+  renderConditions() {
+    return (
+      <div>
```
Comment (evanpurkhiser) [inline static/app/views/alerts/details/sidebar.tsx:24]: Does it need to be in the div?

File: static/app/views/alerts/details/sidebar.tsx [hunk 2]
```diff
@@ -113,6 +184,67 @@ const StatusContainer = styled('div')`
...
+
+const StepContainer = styled('div')`
+  display: flex;
+  align-items: flex-start;
+  flex-grow: 1;
+`;
+
+const StepContent = styled('div')``;
+
+const StepConnector = styled('div')`
```
Comment (evanpurkhiser) [inline static/app/views/alerts/details/sidebar.tsx:204]: Can this just be a pseudo :before element? cc @robinrendle

File: static/app/views/alerts/details/sidebar.tsx [hunk 3]
```diff
@@ -17,6 +19,75 @@ type Props = {
...
+              <div>
+                <StepLead>
+                  {tct('[when:When] [selector] of the following happens', {
+                    when: <Badge />,
+                    selector: 'any',
+                  })}
+                </StepLead>
+                <ConditionsBadge>{'A new issue is created'}</ConditionsBadge>
+                <ConditionsBadge>
+                  {'Issue has more than 100 events in 1 min'}
```
Comment (scttcper) [inline static/app/views/alerts/details/sidebar.tsx:41]: i think this data should be available via the rule passed in via the props

File: static/app/views/alerts/details/sidebar.tsx [hunk 4]
```diff
@@ -17,6 +19,75 @@ type Props = {
...
+                <IconChevron isCircled color="gray200" direction="right" size="sm" />
+              </ChevronContainer>
+              <StepContent>
+                <StepLead>
+                  {tct('[then:Then] perform these actions', {
+                    then: <Badge />,
+                  })}
+                </StepLead>
+                <ConditionsBadge>
+                  {'Send notification to Slack #workflow channel'}
```
Comment (scttcper) [inline static/app/views/alerts/details/sidebar.tsx:78]: these should be available via rule.actions

**Comment by scttcper** [comment]
can we get rid of the dotted line for the last item 
image

**Comment by scttcper** [comment]
image

hmmm not sure what we should do if there are no conditions

**Review by scttcper — APPROVED** [review]

**Pull request merged**